Hybrid projects preview (#7520)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

	- External metadata ingestion allows you to connect directly to your data warehouse, giving you visibility into tables, views, and other resources that aren't defined in dbt. 
- **New**: [dbt Canvas is now generally available](https://getdbt.com/blog/dbt-canvas-is-ga) (GA). Canvas is the intuitive visual editing tool that enables anyone to create dbt models with an easy to understand drag-and-drop interface. Read more [about Canvas](/docs/cloud/canvas) to begin empowering your teams to build more, faster! 
- **New**: [State-aware orchestration](/docs/deploy/state-aware-about) is now in beta! Every time a new job in Fusion runs, state-aware orchestration automatically determines which models to build by detecting changes in code or data.

> - **New**: With Hybrid projects, your organization can adopt complementary dbt Core and dbt Cloud workflows and seamlessly integrate these workflows by automatically uploading dbt Core artifacts into dbt Cloud. [Hybrid projects](/docs/deploy/hybrid-projects) is now available as a preview. 

Author: ```suggestion
- **New**: With Hybrid projects, your organization can adopt complementary dbt Core and dbt Cloud workflows and seamlessly integrate these workflows by automatically uploading dbt Core artifacts into dbt Cloud. [Hybrid projects](/docs/deploy/hybrid-projects) is now available as a preview to [<Constant name="cloud" /> Enterprise accounts](https://www.getdbt.com/pricing). 
```

- **New**: Copilot chat is an interactive interface within the Studio IDE where you can generate SQL code from natural language prompts and ask analytics-related questions. It integrates contextual understanding of your dbt project and assists in streamlining SQL development. For more information, see [Copilot chat](/docs/cloud/copilot-chat-in-studio).
- **New**: Leverage dbt Copilot to generate SQL queries in [Insights](/docs/explore/dbt-insights) from natural language prompts, enabling efficient data exploration within a context-aware interface.
- **New**: The dbt platform Cost management dashboard is now available as a preview for Snowflake users on Enterprise and Enteprise Plus plans. Gain valuable insights into your warehouse spend with the comprehensive and interactive dashboard. Read more [about it](/docs/cloud/cost-management) to get started with your cost savings analysis today!

> - **New**: Apache Iceberg and support for external Iceberg catalogs is now available! External catalogs are a vital part of dbt Mesh and a critical component for supporting large data sets across a variety of warehouses. Read more about [Iceberg](/docs/mesh/iceberg/apache-iceberg-support) and [external catalog](/docs/mesh/iceberg/about-catalogs) support to begin enhancing your dbt Mesh configurations.

Reviewer: If Apache Iceberg and support for external Iceberg catalogs are two separate things, then it should be "Apache Iceberg and support for external Iceberg catalogs are now available!"

But if this sentence wants to convey that support for Apache Iceberg and external Iceberg catalogs is now available, then it should be: "Support for Apache Iceberg and external Iceberg catalogs is now available!"

Reviewer 2: ```suggestion
- **New**:   Apache Iceberg catalog integration support is now available on Snowflake and BigQuery! This is essential to making your dbt Mesh interoperable across platforms, built on Iceberg.  Read more about [Iceberg](/docs/mesh/iceberg/apache-iceberg-support) to begin creating Iceberg tables. 
```

- **New**: Copilot chat is an interactive interface within the Studio IDE where you can generate SQL code from natural language prompts and ask analytics-related questions. It integrates contextual understanding of your dbt project and assists in streamlining SQL development. For more information, see [Copilot chat](/docs/cloud/copilot-chat-in-studio).
- **New**: Leverage dbt Copilot to generate SQL queries in [Insights](/docs/explore/dbt-insights) from natural language prompts, enabling efficient data exploration within a context-aware interface.
- **New**: The dbt platform Cost management dashboard is now available as a preview for Snowflake users on Enterprise and Enteprise Plus plans. Gain valuable insights into your warehouse spend with the comprehensive and interactive dashboard. Read more [about it](/docs/cloud/cost-management) to get started with your cost savings analysis today!

> - **New**:   Apache Iceberg catalog integration support is now available on Snowflake and BigQuery! This is essential to making your dbt Mesh interoperable across platforms, built on Iceberg.  Read more about [Iceberg](/docs/mesh/iceberg/apache-iceberg-support) to begin creating Iceberg tables. 

Reviewer: ```suggestion
- **New**: Apache Iceberg catalog integration support is now available on Snowflake and BigQuery! This is essential to making your dbt Mesh interoperable across platforms, built on Iceberg. Read more about [Iceberg](/docs/mesh/iceberg/apache-iceberg-support) to begin creating Iceberg tables. 
```
